.github/workflows/Dockerfile

```diff
@@ -50,6 +50,7 @@ RUN    apt-get update              \
         libsecp256k1-dev           \
         libsqlite3-dev             \
         libssl-dev                 \
+        libunwind-dev              \
```

Comment:
This package needs to be added to the README: https://github.com/runtimeverification/k/blob/0ca007c930b50e4d70fc82aa1ff18b707e8362fe/README.md?plain=1#L79-L105